docs/csharp/programming-guide/types/index.md

```diff
@@ -63,7 +63,7 @@ ms.author: "wiwagn"
  C# provides a standard set of built-in numeric types to represent integers, floating point values, Boolean expressions, text characters, decimal values, and other types of data. There are also built-in `string` and `object` types. These are available for you to use in any C# program. For more information about the built-in types, see [Reference Tables for Types](../../../csharp/language-reference/keywords/reference-tables-for-types.md).  
   
 ## Custom Types  
- You use the [struct](../../../csharp/language-reference/keywords/struct.md), [class](../../../csharp/language-reference/keywords/class.md), [interface](../../../csharp/language-reference/keywords/interface.md), and [enum](../../../csharp/language-reference/keywords/enum.md) constructs to create your own custom types. The .NET Framework class library itself is a collection of custom types provided by Microsoft that you can use in your own applications. By default, the most frequently used types in the class library are available in any C# program. Others become available only when you explicitly add a project reference to the assembly in which they are defined. After the compiler has a reference to the assembly, you can declare variables (and constants) of the types declared in that assembly in source code. For more information, see [.NET Framework Class Library](http://go.microsoft.com/fwlink/?LinkID=217856).  
+ You use the [struct](../../../csharp/language-reference/keywords/struct.md), [class](../../../csharp/language-reference/keywords/class.md), [interface](../../../csharp/language-reference/keywords/interface.md), and [enum](../../../csharp/language-reference/keywords/enum.md) constructs to create your own custom types. The .NET Framework class library itself is a collection of custom types provided by Microsoft that you can use in your own applications. By default, the most frequently used types in the class library are available in any C# program. Others become available only when you explicitly add a project reference to the assembly in which they are defined. After the compiler has a reference to the assembly, you can declare variables (and constants) of the types declared in that assembly in source code. For more information, see [.NET Framework Class Library](/dotnet/api/?view=netframework-4.7.1).  
```

Comment:
Given the difference between the MSDN top-level page and the API Browser, I think the best replacement for this FWLink is the .NET Framework Class Library Overview at https://docs.microsoft.com/en-us/dotnet/standard/class-library-overview. That page has a link to the class library. Right now, it's a FWLink, but I'll open a PR now to change it to the API Browser.

Reply:
@rpetrusha that's right. In that case I'll also omit "Framework" from the link text: ".NET Framework Class Library" => ".NET Class Library Overview" (to match your PR). Then, I don't see a reason to mention Framework at all in the `types/index.md`. So, I'll omit all Framework mentions from this document (I guess there is nothing specific to .NET Framework in the types overview). What do you think of that?

Reply:
That sounds good, @pkulikov.Thanks.